internal/ingress/controller/template/template_test.go

```diff
@@ -1165,7 +1165,7 @@ func TestBuildOpenTracing(t *testing.T) {
 
 	cfgJaeger := config.Configuration{
 		EnableOpentracing:   true,
-		JaegerCollectorHost: "jaeger-host.com",
+		JaegerCollectorHost: "localhost",
```

Comment:
What do you think to use: `jaeger.lvh.me`? It would test  resolving not just by `/etc/hosts`, but also use DNS resolving:

```
nslookup jaeger.lvh.me     
Server:		2003:fb:ef09:9d00::1
Address:	2003:fb:ef09:9d00::1#53

Non-authoritative answer:
Name:	jaeger.lvh.me
Address: 127.0.0.1
```

Reply:
I made this change, should pass tests fine. I don't think it's necessarily important to test that resolvers themselves work here, but why not match production as much as possible?

Reply:
Thank you for the changes.

> why not match production as much as possible?
I think a general deployment of ingress and collector will be deployed to different node pools.
Discovery would be through the kubernetes services, that would use resolving through nameservers.